sinks/gcm/driver.go

```diff
@@ -94,11 +94,11 @@ func getDescription(metric sink_api.MetricDescriptor) string {
 	}{
 		{
 			name:        "uptime",
-			description: "Rate of change of time since start in milliseconds per second",
```

Comment:
We just turned all cumulative into rates. I don't think it's easy to use either way... Maybe keeping cumulative is less worse